qiskit/providers/aer/noise/device/models.py

```diff
@@ -134,8 +134,11 @@ def basic_device_readout_errors(properties):
     """
     errors = []
     for qubit, value in enumerate(readout_error_values(properties)):
-        if value is not None and value > 0:
-            probabilities = [[1 - value, value], [value, 1 - value]]
+        if value is not None:
+            if isinstance(value, list) and len(value) == 2:
```

Comment:
With the change to make both error cases return a pair of readout error values this can simplified (and adding check that values aren't zero):
```python
if value is not None and not np.allclose(value, [0, 0]):
    probabilities = [[1 - value[0], value[0]], [value[1], 1 - value[1]]]
```

Reply:
Done

Reply:
Done